src/main/cms/cms_menu_misc.c

```diff
@@ -127,17 +127,15 @@ CMS_Menu cmsx_menuRcPreview = {
     .entries = cmsx_menuRcEntries
 };
 
-static uint16_t motorConfig_minthrottle;
-static uint8_t motorConfig_digitalIdleOffsetValue;
+static uint8_t motorConfig_motorIdle;
 static uint8_t rxConfig_fpvCamAngleDegrees;
 static uint8_t mixerConfig_crashflip_rate;
 
 static const void *cmsx_menuMiscOnEnter(displayPort_t *pDisp)
 {
     UNUSED(pDisp);
 
-    motorConfig_minthrottle = motorConfig()->minthrottle;
-    motorConfig_digitalIdleOffsetValue = motorConfig()->digitalIdleOffsetValue / 10;
+    motorConfig_motorIdle = motorConfig()->motorIdle / 10;
```

Comment:
```suggestion
    motorConfig_motorIdle = motorConfig()->motorIdle;
```

Reply:
I wasn't intending to change the user experience in the goggles.  
Previously if the dshot idle was 5.5%, the CLI number was 550.
The goggles use would see 55 from
```
motorConfig_motorIdle = motorConfig()->motorIdle / 10;
```
And when written back, motorConfig_motorIdle needed to be multiplied by 10 to make up for the incoming division.
I wasn't intending to change the current OSD behaviour.

Reply:
It's true that the old approach would cause the loss of the second decimal part of the idle value, if configured, but that hasn't been an issue in the past.
I personally would like Configurator GUI to only show one decimal point in the percentage being set.  We don't need 2 decimal points of accuracy.

Reply:
https://github.com/betaflight/betaflight-configurator/pull/4196/commits/b114648f7717eababa1b3e8b17afa9dc402f9cee